### PR TITLE
fix doc of detection's img_info param

### PR DESCRIPTION
**Pull request by FDInSky**
fix doc of detection's img_info param based on the issue https://github.com/PaddlePaddle/PaddleDetection/issues/203

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]

File: python/paddle/fluid/layers/detection.py [hunk 1]
```diff
@@ -2748,8 +2748,8 @@ def generate_proposals(scores,
             represents the differece between predicted box locatoin and
             anchor location. The data type must be float32.
         im_info(Variable): A 2-D Tensor with shape [N, 3] represents origin
-            image information for N batch. Info contains height, width and scale
-            between origin image size and the size of feature map.
+            image information for N batch. Height and width is the input size
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/detection.py]: is -> are, size -> sizes

File: python/paddle/fluid/layers/detection.py [hunk 2]
```diff
@@ -2842,7 +2842,7 @@ def box_clip(input, im_info, name=None):
             the last dimension is 4 and data type is float32 or float64.
         im_info(Variable): The 2-D Tensor with shape [N, 3] with layout 
             (height, width, scale) represeting the information of image. 
-            height and width is the input size and scale is the ratio of input
+            Height and width is the input size and scale is the ratio of network input
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/detection.py]: is -> are, size -> sizes

File: python/paddle/fluid/layers/detection.py (hunk 1 above)
Comment (FDInSky) [reply python/paddle/fluid/layers/detection.py]: 修改了

File: python/paddle/fluid/layers/detection.py (hunk 2 above)
Comment (FDInSky) [reply python/paddle/fluid/layers/detection.py]: 修改了

**Review by Boyan-Liu — APPROVED** [review]

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**